### PR TITLE
Add colour differentiation and examples for more commands 

### PR DESCRIPTION
**Pull request by moisterrific**
https://github.com/Pryaxis/TShock/issues/2418

### CODE REVIEW

**Comment by moisterrific** [comment]
correction: https://github.com/Pryaxis/TShock/pull/2419/commits/e88ae85683673a2f298503315ce67d18a9d330bd is for Firework command, not Rocket

File: TShockAPI/Commands.cs [hunk 1]
```diff
@@ -5578,94 +5663,115 @@ private static void Clear(CommandArgs args)
...
-							float dX = Main.item[i].position.X - args.Player.X;
-							float dY = Main.item[i].position.Y - args.Player.Y;
+							float dX = Main.item[i].position.X - user.X;
+							float dY = Main.item[i].position.Y - user.Y;
 
 							if (Main.item[i].active && dX * dX + dY * dY <= radius * radius * 256f)
 							{
 								Main.item[i].active = false;
-								TSPlayer.All.SendData(PacketTypes.ItemDrop, "", i);
+								everyone.SendData(PacketTypes.ItemDrop, "", i);
```
Comment (hakusaro) [inline TShockAPI/Commands.cs:5683]: I get the feeling you want to be a ruby developer but don't know it

Comment (moisterrific) [reply TShockAPI/Commands.cs:5683]: idk what u mean lmao

**Comment by moisterrific** [comment]
This doesn't cover all commands btw, I'll probably add those later.

**Review by QuiCM — DISMISSED** [review]

**Review by hakusaro — DISMISSED** [review]

**Review by hakusaro — APPROVED** [review]

**Pull request merged**